docs/docs/migration-guide.mdx

```diff
@@ -13,6 +13,35 @@ This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
 
+## Rasa 3.2 to 3.3
+
+### ConcurrentRedisLockStore
+
+The [`ConcurrentRedisLockStore`](./lock-stores.mdx#concurrentredislockstore) is the new Redis implementation of the abstract interface `LockStore`.
+This implementation is able to handle messages received in parallel for the same conversation id by different Rasa instances.
+
+In order to switch from using the `RedisLockStore` to the `ConcurrentRedisLockStore`, specify the parameter (new in 3.3) `concurrent_mode` as `true` in `endpoints.yml`:
+
+```yaml-rasa
+lock_store:
+    type: "redis"
+    concurrent_mode: true
+    url: <url of the redis instance, e.g. localhost>
+    port: <port of your redis instance, usually 6379>
+    password: <password used for authentication>
+    db: <number of your database within redis, e.g. 0>
+    key_prefix: <alphanumeric value to prepend to lock store keys>
+```
+
+Note that not specifying the `concurrent_mode` parameter or setting it to `false` will default to using the `RedisLockStore` implementation.
+
+No database migration is required when switching to the `ConcurrentRedisLockStore`, and you may use the same Redis database number.
+You may want to delete all the preexisting keys if using the same Redis database number.
```

Comment:
Why may the user want to do that? Will it influence the behaviour in any way, or is it more to clear out room for storage?

Reply:
It's just to clear out room, because the old keys won't be used any longer, they become obsolete.

Reply:
Makes sense, maybe make that connection explicit here?

Reply:
Done: https://github.com/RasaHQ/rasa/pull/11342/commits/2ad5a5a6ce35c52c8476ceceb6756ff117b86051